features/seed.py

```
@@ -0,0 +1,633 @@
+def _link_dir(source: str, destination: str) -> None:
+    """Point `destination` at `source` without copying it.
+
+    A directory symlink on Windows needs either administrator rights or
+    Developer Mode, neither of which a hosted runner offers, so it gets a
+    junction instead - `mklink /J` is a cmd builtin and needs no privileges on
+    NTFS. Everywhere else a symlink does.
+    """
+    if platform.system() == "Windows":
+        # normpath again rather than trusting the caller: a single forward slash
+        # anywhere in these makes cmd read the rest of the path as switches.
+        result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)